cmd/waku/server/rest/health.go

```diff
@@ -0,0 +1,48 @@
+package rest
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/waku-org/go-waku/waku/v2/node"
+)
+
+type HealthService struct {
+	node *node.WakuNode
+	mux  *chi.Mux
+}
+
+const routeHealth = "/health"
+
+func NewHealthService(node *node.WakuNode, m *chi.Mux) *HealthService {
+	h := &HealthService{
+		node: node,
+		mux:  m,
+	}
+
+	m.Get(routeHealth, h.getHealth)
+
+	return h
+}
+
+type HealthResponse string
+
+func (d *HealthService) getHealth(w http.ResponseWriter, r *http.Request) {
+	isReady, err := d.node.RLNRelay().IsReady(r.Context())
```

Comment:
Is just checking health of relay sufficient?
If services are enabled, we should also include those in this healthcheck endpoint.
If someone builds a TUI/GUI later for serviceNode then this endpoint would be useful in fetching and displaying health status of each service.

Reply:
For now this is aligned with the behavior from nwaku

Reply:
Maybe we can open an enhancement for this task then. Would be better to have something like this.

Reply:
Tracking work for this in https://github.com/waku-org/go-waku/issues/740